src/botcity/maestro/interfaces/artifact.ts

```diff
@@ -0,0 +1,51 @@
+interface Content {
+  id: number
+  type: string
+  taskId: number
+  name: string
+  fileName: string
+  organizationId: number
+  dateCreation: string
+  taskName: string
+}
+
+interface Sort {
+  sorted: boolean
+  unsorted: boolean
+  empty: boolean
+}
+
+interface Pageable {
+  sort: Sort
+  pageNumber: number
+  pageSize: number
+  offset: number
+  paged: boolean
+  unpaged: boolean
+}
```

Comment:
As a TODO we should look to move the common types such as `Pageable` to a `commons.ts` or some other name.

Reply:
As this implementation is simple was inserted inside this pull request.